src/pages/faq.tsx

```diff
@@ -1,33 +1,143 @@
+        voting for is slashed. Together, this means that no principal is at risk under normal
+        operating circumstances when using StakedCelo.
+      </FaqItem>
+      <FaqItem question="What are Epoch Rewards and how do they impact StakedCelo?">
+        Celo <EpochRewardsLink /> are similar to block rewards in other blockchains and used to
+        create several types of incentives which help to keep the blockchain working securely. Among
+        others, CELO holders who lock their tokens into{' '}
+        <Link url="https://docs.celo.org/celo-codebase/protocol/proof-of-stake/epoch-rewards/locked-gold-rewards">
+          Locked CELO
+        </Link>
+        receive rewards for active participation, including voting for groups that elected
+        validators. The StakedCelo protocol receives such rewards as it stakes CELO into Locked
+        CELO.
+        <br />
+        <br />
+        These rewards are{' '}
+        <Link url="https://docs.celo.org/celo-codebase/protocol/proof-of-stake/epoch-rewards/locked-gold-rewards#introduction-to-locked-celo-rewards">
+          automatically compounded
+        </Link>{' '}
+        at the Celo protocol level. The rewards accruing in the StakedCelo protocol depend on
+        several factors, including participants and events. For example, the validator groups that
+        StakedCelo votes for and the behavior of their validators (the Slashing Penalty and the
+        Uptime Score influence the distributed tokens), the total amount of CELO staked (the reward
+        rate is adjusted if the number of staked CELO deviates from the target) or Celo governance
+        (Celo governance can vote to change the underlying mechanism fundamentally). More details
+        can be found in the Celo docs on <EpochRewardsLink />.
+      </FaqItem>
+      <FaqItem question="Why is 1 stCELO not equal 1 CELO?">
+        stCELO is an ERC-20 compatible non-rebasing token. This means that the amount of tokens a
+        user has in their wallet stays constant over time if that user takes no actions.
+        Consequently, one stCELO is increasing in value over time (measured in CELO), as epoch
+        rewards are being accrued in the protocol. Note that this only describes the value of stCELO
+        and CELO when being staked and unstaked via the StakedCelo protocol. Prices on decentralized
+        exchanges are subject to market conditions, market fluctuations, and not per se predictable.
+      </FaqItem>
+      <FaqItem question="What is the difference between StakedCelo and stCELO?">
+        StakedCelo is the name of the protocol and stCELO is the ERC20 token users receive when
+        staking their CELO with the protocol.
+      </FaqItem>
+      <FaqItem question="How is StakedCelo secure?">
+        StakedCelo has been audited and the reports are published{' '}
+        <Link url="https://github.com/celo-org/staked-celo/tree/soloseng/add-audit-report">
```

Comment:
swithch link to master branch

https://github.com/celo-org/staked-celo/tree/master/audit

Reply:
Done